nucleus/test_launch_integration.py

```diff
@@ -0,0 +1,74 @@
+import io
+from typing import Any, Callable, Dict
+
+from PIL import Image, ImageDraw
+
+# From scaleapi/server/src/lib/select/api/types.ts
+_TOP_LEVEL_REQUIRED_KEYS = {"geometry", "type"}
+_TOP_LEVEL_OPTIONAL_KEYS = {"label", "confidence", "classPdf", "metadata"}
+# TODO idk which ones are right for nucleus
```

Comment:
TODO: I'm not sure which keys make sense to include, I've semi-arbitrarily included these 4 for now